src/triage/component/catwalk/storage.py

```diff
@@ -496,14 +566,17 @@ def load_metadata(self):
     def save(self):
         raise NotImplementedError
 
+    def clear_cache(self):
+        self.design_matrix = None
+        self.labels = None
+        self.metadata = None
+
     def __getstate__(self):
         """Remove object of a large size upon serialization.
 
         This helps in a multiprocessing context.
         """
-        self.matrix = None
-        self._labels = None
-        self.metadata = None
+        self.clear_cache()
```

Comment:
not a big deal but it seems nicer to exclude the cache from the state, rather than affect the data of the instance you're trying to pickle.

Reply:
Makes sense. This was more useful when this was the only method of cache busting (and we didn't want it to get loaded in two different processes), but now that we have a more direct method of controlling the cache simply excluding the cache from the state seems better.